src/Auth0.php

```diff
@@ -544,6 +544,42 @@ public function setRefreshToken($refreshToken)
         return $this;
     }
 
+    /**
+     * Renews the access token and ID token using an existing refresh token.
+     * 
+     * @throws CoreException If the Auth0 object does not have access token and refresh token
+     * @return bool
+     */
+    public function renewTokens()
+    {
+        if (!$this->accessToken) {
+            throw new CoreException('Can\'t renew the access token if there isn\'t one valid');
+        }
+
+        if (!$this->refreshToken) {
+            throw new CoreException('Can\'t renew the access token if there isn\'t a refresh token available');
+        }
+
+        $response = $this->authentication->oauth_token([
+            'grant_type' => 'refresh_token',
+            'client_id' => $this->clientId,
+            'client_secret' => $this->clientSecret,
+            'refresh_token' => $this->refreshToken,
+        ]);
+
+        if (empty($response['access_token']) || empty($response['id_token'])) {
+            return FALSE;
```

Comment:
I would `throw` an `ApiException` here instead of returning `false`. See the `exchange()` method for an example.

Reply:
Agreed. Also the method may become `void` as the return value will never be `FALSE`.